pyintelowl/cli/analyse.py

```diff
@@ -32,22 +32,10 @@
     """,
     ),
     click.option(
-        "-fp",
-        "--force-privacy",
+        "-t",
+        "--tlp",
         is_flag=True,
```

Comment:
Should `is_flag` be True ? AFAIK, this should be used only for boolean yes/no options.

Reply:
@sp35

Reply:
We should use the `click.Choice` here. (https://click.palletsprojects.com/en/8.0.x/options/#choice-options)

Reply:
ya, right